### ISSUE TITLE
Where to get the training/test set of ACAS Xu?

### ISSUE THREAD

**Issue by dgl-prc** [rnbguy/acasxu_tf_keras#1]
Hi, 
I wanna train a model of ACAS Xu by myself but I can't find the training set. How do you train the tf model of ACAS Xu and where to get the training set?

**Comment by rnbguy** [comment]
ACAS Xu training/test sets are not publicly available. You can refer to this issue here - NeuralNetworkVerification/Marabou#173

**Comment by dgl-prc** [comment]
Thank you!
